cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -277,4 +276,44 @@ export class MigrationWave {
         cy.contains("li", date.toLocaleString("en-us", { month: "long" })).click();
         cy.contains("button:not([disabled])", `${date.getDate()}`).first().click();
     }
+
+    public clickOnApplicationStatus() {
```

Comment:
This column is actually related to wave status and not application status. So, this method should be renamed to 'clickWaveStatus'